test/parallel/test-child-process-spawn-typeerror.js

```diff
@@ -35,10 +35,14 @@ const invalidArgValueError =
 const invalidArgTypeError =
   common.expectsError({ code: 'ERR_INVALID_ARG_TYPE', type: TypeError }, 12);
```

Comment:
Can we simply change the second argument to 13 here and reuse `invalidArgTypeError` instead of defining `invalidArgTypeErrorCallsOnce`?

Reply:
👍 yes of course.